src/app/components/lanes-child/lanes-child.component.scss

```diff
@@ -0,0 +1,29 @@
+.pending{
+    background-color: white;
+    height: 135px;
+    border: 1px solid #E0E0E0;
+    box-sizing: border-box;
+    border-radius: 5px;
+}
+p{
```

Comment:
@SwathiChalumuri We shouldn't give styles directly to tag as it may effect in other parts of the application wherever we  use those tags. It's good to write a class and assign class to that tag

Reply:
ok